docs/sample_notebooks/visualize-mosaic.ipynb

```diff
@@ -0,0 +1,429 @@
+{
```

Comment:
<p>if it's MAAP specific, should it go under MAAP documentation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/stac_ipyleaflet/pull/115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Valid point, makes me wonder if we just need to find a more publicly available mosaic example for these docs. Agreed this specific example should move to MAAP docs. cc: @grallewellyn

Reply:
I was working on replacing mentions of ipycmc in the docs with stac_ipyleaflet, so I will make sure this gets added to maap-documentation. Thank you so much!

Reply:
https://github.com/MAAP-Project/maap-documentation/pull/333

Reply:
> Valid point, makes me wonder if we just need to find a more publicly available mosaic example for these docs.

That would be great - and we could use one of the URLs in the `raster` DIY notebook too, which is also MAAP-specific, I realize now.


And we also probably need to remove the blocks about deriving the asset path from bucket variable and environment variables, which are relative to the environment...